cassiopeia/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: "3"
+services:
+
+  apache:
+    container_name: cassie-apache
+    image: ${ENV_APACHE_CONTAINER} 
+    ports:
+      - "80:80"
+      - "443:443"
+    environment:
+      - ENV_APACHE_SERVER_NAME=${ENV_APACHE_SERVER_NAME}
+    volumes:
+      - ${ENV_REACT_APPDIR}/build:/var/www/html
+      - ${ENV_SSL_CERT_DIR}:/certs
+    networks:
+      local:
+        aliases:
+          - cassie-apache
+
+  mongodb:
+    container_name: cassie-mongodb
+    image: mongo:4.0
+    environment:
+       - MONGO_DATA_DIR=/data/db
+       - MONGO_LOG_DIR=/dev/null
+    volumes:
+       - mongodbdata:/data/db 
+    ports:
+      - "127.0.0.1:${ENV_MONGO_PORT:-27017}:27017"
+    command: mongod 
+    networks:
+      local:
+        aliases:
+          - cassie-mongodb
+
+  spring:
+    container_name: cassie-spring
+    build: ./spring
+    ports:
+      - "${ENV_APACHE_TOMCAT_PORT:-3030}:3030"
+    volumes:
+      - "${ENV_SPRING_BOOT_APPDIR}:/home/gradle"
+    environment:
+      SPRING_BOOT_ENVIRONMENT: ${ENV_DOCKER_ENVIRONMENT}
+    privileged: true
+    networks:
+      local:
+        aliases:
+          - cassie-spring
+
+volumes:
```

Comment:
Is this volumes configuration in the docker-compose.yml complete?  It looks blank.

Reply:
I know it looks odd...but this is again taken from orion and it works as is, so I think we are ok.

Reply:
Works for me!

This was my only concern with the pull request, so I'll mark it reviewed.